actors/runtime/runtime.go

```diff
@@ -231,3 +231,30 @@ type StateHandle interface {
 	// ```
 	StateTransaction(obj cbor.Er, f func())
 }
+
+type Pricelist interface {
```

Comment:
I don't think this should be visible to actor code that imports `runtime`. Move it and `GasCharge` to the `vm` package.